### DISCUSSION TITLE
CommandBarButton how to set icon and label size？

### DISCUSSION THREAD

**Discussion by gatspy** [amwx/FluentAvalonia#431] [Q&A]
As shown in the picture below, the text display is incomplete.
<img width="259" alt="image" src="https://github.com/amwx/FluentAvalonia/assets/804446/c2c5d704-4519-4aad-9f67-b64d72971a8e">

**Comment by timunie** [comment]
you may need to adjust the style acc. to your needs. lookup original styles: https://github.com/amwx/FluentAvalonia/blob/fb7a7c2aa2d490e48f137c90b82e36c3d0c8a3da/src/FluentAvalonia/Styling/ControlThemes/FAControls/CommandBar/CommandBarButtonStyles.axaml#L66C47-L66C47
